### ISSUE TITLE
Handle broken json for playerstats/update_log.json

### ISSUE THREAD

**Issue by JabLuszko** [Map-A-Droid/MAD#774]
From time to time there is a guy with an issue about broken json in playerstats or update_log.json. Most of the times it happens when they run out of the disk space and saving file fails.
We could handle those and show an error with some description (or nuke file if not important and move on).

Example backtrace from:
```
[03-15 19:37:12.37] [          scanner] [    WebsocketServer:82  ] [    INFO] Starting websocket-server...
Traceback (most recent call last):
  File "start.py", line 235, in <module>
    device_Updater = deviceUpdater(ws_server, args, jobstatus, db_wrapper)
  File "/root/MAD/mapadroid/utils/updater.py", line 58, in __init__
    self._log = json.load(logfile)
  File "/usr/local/lib/python3.7/json/__init__.py", line 296, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/local/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/managers.py", line 195, in handle_request
    result = func(c, *args, **kwds)
  File "/usr/lib/python3.6/multiprocessing/managers.py", line 359, in create
    obj = callable(*args, **kwds)
  File "/home/ubuntu/MAD/mitm_receiver/MitmMapper.py", line 42, in __init__
    self.__playerstats[origin].open_player_stats()
  File "/home/ubuntu/MAD/mitm_receiver/PlayerStats.py", line 71, in open_player_stats
    data = json.load(f)
  File "/usr/lib/python3.6/json/__init__.py", line 299, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**EDIT:** If someone searching for help would find this - current way to recover MAD is to delete the brokne file. If your log contains "**_PlayerStats_**" then delete *.json files in files/ directory. If your log contains "_**deviceUpdater**_" then delete update_log.json from main MAD directory.

**Comment by crhbetz** [comment]
I created a fix #776 for the first example. 
The second one should already be handled for quite some time. (your log excerpt has to be kinda old with the `mapadroid` folder not yet present 😃 )

**Comment by krzee** [comment]
maybe the second log excerpt, but the first one has mapadroid subdir, and is fully up to date with git. the file update_log.json existing but being empty is enough to stop madmin from loading. removing the file fixes it

**Comment by JabLuszko** [comment]
Could be outdated. I've copy-pasted both from Discord - I remember those existing, never really paid attention if fixed, my bad, sorry. Wanted to group it into one - I was more thinking some 'wrapper class' to json and fix both in one go, but #776 will work too :)
